README.md

```diff
@@ -20,11 +20,23 @@ git submodule update --init --recursive
 
 # Install dependencies
 npm install
+
+# Set specified versions for submodules
+npm run set-versions
```

Comment:
Please write in readme how where we should specify versions of different reposiories

Reply:
Done